pkgs/applications/misc/stretchly/default.nix

```diff
@@ -29,7 +29,7 @@ stdenv.mkDerivation rec {
     runHook preInstall
 
     mkdir -p $out/bin $out/share/${pname}/
-    mv resources/app.asar $out/share/${pname}/
+    mv resources/app.asar* $out/share/${pname}/
```

Comment:
The new version contains `resources/app.asar.unpacked/node_modules` which also need to be copied. Or `electron` will raise an error about file not found.